app.js

```diff
@@ -179,6 +220,27 @@ sitemap(config, (sitemap, urls) => {
 	console.log('Sitemap %s registered %s / %s urls', sitemap.url, results.length, urls.length);
 });
 
+const pingbackUrl = (data, cb) => {
+	//Process pingBack
+	if (pingback.ok) {
+		pingback.send({result: data.result, action: data.action, url: data.url}, (data2) => {
+			if (data2.ok) {
```

Comment:
Why not directly pass data as 1st parameter ?

Reply:
Data is cleaned to pass only relevant information to the pingback lib (strip callback)